rest/models/LastVisitedPage.go

```diff
@@ -0,0 +1,7 @@
+package models
+
+type LastVisitedPage struct {
+  BaseModel
+  Pathname string
```

Comment:
We also have to add the `json` definition. So go knows how to marshal/unmarshal JSON

Reply:
Should I add that json definition to all of the properties? 
so, i.e: Pathname string `json: pathname` and so on for the rest?

Reply:
Yeah